js/src/components/lineage/SeverDisconnectedModalContent.tsx

```diff
@@ -29,19 +29,48 @@ export function ServerDisconnectedModalContent({ connect }: { connect: () => voi
   );
 }
 
-export function RecceShareInstanceDisconnectedModalContent({ shareUrl }: { shareUrl: string }) {
+export function RecceInstanceDisconnectedModalContent({
+  shareUrl,
+  mode,
+}: {
+  shareUrl: string;
+  mode: "read only" | "preview mode";
+}) {
+  const contents = {
+    "read only": {
+      title: "Share Instance Expired",
+      body: "This Share Instance has expired. Please restart the share instance.",
+      action: "Restart",
+      link: shareUrl,
+    },
+    "preview mode": {
+      title: "Preview Instance Expired",
+      body: "This Preview Instance has expired. To browse more, please book a meeting with us.",
+      action: "Contact us",
+      link: "https://cloud.datarecce.io/",
```

Comment:
Should we link to our landing page instead of cloud page?
ex: https://reccehq.com/demo/